deepchem/feat/graph_features.py

```diff
@@ -166,6 +166,9 @@ def atom_features(atom, bool_id_feat=False, explicit_H=False):
                              [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]) + \
               one_of_k_encoding_unk(atom.GetImplicitValence(), [0, 1, 2, 3, 4, 5, 6]) + \
               [atom.GetFormalCharge(), atom.GetNumRadicalElectrons()] + \
+              one_of_k_encoding_unk(atom.GetChiralTag(),
```

Comment:
can we put this behind a feature flag at least to start?  
```python
use_chirality=False
```

Reply:
We shouldn't change the model default behaviour until we have reasonable proof that it does better on average for datasets we care about.

Reply:
Yes, I can try that! Sorry for failing the test, I actually tested on my repo using Theano, it works fine. I didn't expect so tricky deepchem is!

Reply:
Let me fix it